src/index.css

```diff
@@ -9,6 +9,7 @@
     --color-fg-variant: rgb(60% 60% 60%);
     --color-border: rgb(86% 86% 86%);
     --color-accent: rgb(39.22% 42.35% 100%);
+    --color-footer-bg: #f0f0f0;
```

Comment:
We may want to use this elsewhere. Perhaps we call it `—color-bg-variant` so the variable name isn’t coupled to a particular element?

And I’ll leave you to decide if you want to change it to `rgb(94.12% 94.12% 94.12%)` for consistency.

Reply:
I can make those changes probably not a bad idea.